cli/src/commands/download.ts

```diff
@@ -48,6 +48,12 @@ export default class Download extends Command {
     await this.authenticate();
     let application: Application | undefined;
     try {
+      this.drop(this.lycanClient.reportAnalytics([{
+        type: 'event',
+        category: 'user',
+        action: 'download (get app)',
```

Comment:
Suggestion: Maybe use strings with no parentheses and spaces? Something like `download::getApp`?
Future you might thank you for it one day.

Reply:
Done.